### PR TITLE
Update styling for document history tab

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description

After some discussion with Nikin there are a few styling changes we can make to improve the history tab that is rendered ont the edition summary, edit and edit translation pages.

This PR does the following: 

Uses the GOV.UK Publishing list component in place of custom HTML in the document tab component

And for the audit entry component and editorial remark component:

1. Replaces list tag with a div (the output of this is being passed
into the GOV.UK List component as an item)
2. Makes the document action a H4
3. Updates the ordering of the author and datetime
4. Applies govuk-body-s to the author and datetime
5. Applies $govuk-secondary-text-colour to the datetime (hint text)

### Before

image

### After 

image

## Trello card 

https://trello.com/c/o03t3EYI/1046-fix-styling-of-history-component-based-on-design-feedback

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: app/components/admin/editions/audit_trail_entry_component.html.erb [hunk 1]
```diff
@@ -1,8 +1,13 @@
-<li class="app-view-editions-audit-trail-entry__list-item govuk-!-margin-bottom-4">
+<div class="app-view-editions-audit-trail-entry__list-item govuk-!-margin-bottom-4">
+  <p class="govuk-!-font-weight-bold govuk-!-margin-bottom-0"><%= action %></p>
```
Comment (ollietreend) [inline app/components/admin/editions/audit_trail_entry_component.html.erb:2]: The 'action' is wrapped in a `<p>` tag, but the commit 7fe2b9add13e17136672982b196c364e509b37b2 says it should be a `<h4>`.

Does this need changing? 🤔 

Commit message

**Review by ollietreend — COMMENTED** [review]
This looks good – just one thing which I think might have been missed.

File: app/components/admin/editions/audit_trail_entry_component.html.erb (hunk 1 above)
Comment (davidgisbey) [reply app/components/admin/editions/audit_trail_entry_component.html.erb:2]: oh sorry that just meant to be for a editorial remark. I'll update the commit message now

**Review by ollietreend — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**